src/css/_booking-form.scss

```diff
@@ -1,3 +1,19 @@
+.display-booking-button {
+  display: flex;
```

Comment:
I was able to create a mixin just for flex because I found myself using the same combination over and over again. I've noticed that display: flex, flex-direction: column, and justify-content:center are often paired together.